src/org/gh/Teacher.java

```diff
@@ -2,34 +2,34 @@
 package org.gh;
 
 // extends Parent means we inherit from Parent
-// This gives Teacher the ability to use Parent methods and attributes
+// This gives Advisor the ability to use Parent methods and attributes
```

Comment:
If you rename "Teacher" class to "Advisor" you'll need to also update the file name from Teacher.java to Advisor.java.  That's at least the first error.